integreat_cms/release_notes/current/unreleased/1589.yml

```
@@ -0,0 +1,2 @@
+en: Fix filter reset function
+de: Repariere Filter-Reset-Funktion
```

```suggestion
de: Repariere die "Filter zurücksetzen"-Funktion
```
Found something else :see_no_evil: